zeebe/zb-db/src/main/java/io/camunda/zeebe/db/impl/rocksdb/transaction/RawTransactionalColumnFamily.java

```diff
@@ -140,6 +139,50 @@ private void forEach(
     }
   }
 
+  public static void forEachPreallocated(
```

Comment:
:wrench: Would be nice to document the semantics of these two variants of "forEach". It is not easy to understand the difference from the method name.